demo/index.html

```diff
@@ -11,35 +11,27 @@
 <html>
 <head>
   <title>iron-icon demo</title>
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script type="module" src="../iron-icon.js"></script>
-  <script type="module" src="../../iron-iconset/iron-iconset.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
+  <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+  <script type="module">
+    import '../iron-icon.js';
+    import '@polymer/iron-iconset/iron-iconset.js';
+    import '@polymer/iron-demo-helpers/demo-snippet.js';
+  </script>
 </head>
 <body unresolved>
-  <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<custom-style>
-    <style is="custom-style" include="demo-pages-shared-styles"></style>
-  </custom-style>`;
-
-document.body.appendChild($_documentContainer.content);
-</script>
-
   <script type="module">
-const $_documentContainer = document.createElement('template');
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+
+    const template = document.createElement('template');
+    template.innerHTML = `
```

Comment:
does it not work if you just put this HTML in the document like it originally was?

Reply:
Same issue as [this comment I left on iron-fit-behavior](https://github.com/PolymerElements/iron-fit-behavior/pull/93#discussion_r199027618). :/ I'll ask @azakus about this tomorrow.